Publish cluster UI v21.2.1

nathanstilwell opened the pull request:
This change includes a version bump for a package that has already [been published](https://www.npmjs.com/package/@cockroachlabs/cluster-ui/v/21.2.1) to npm.

Commit:
v21.2.1

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/73562)
<!-- Reviewable:end -->

Commit:
ui: Update cluster-ui README

Updating some language and versions to reflect current publishing and
versioning.

Release note: None
